src/scss/_statements.scss

```diff
@@ -24,184 +24,176 @@
 /**
  * drop
  */
-$drop--selector: selector--constructor($drops--synthax, 'drop');
-
-@include selector($drop--selector) {
-  @include declaration('font-size', $drops--font-size);
-  @include declaration('font-family', $drops--font-family);
-  @include declaration('font-weight', $drops--font-weight);
-  @include declaration('text-transform', $drops--text-transform);
-  @include declaration('letter-spacing', $drops--letter-spacing);
-  @include declaration('line-height', $drops--line-height);
-  @include declaration('text-align', $drops--text-align);
-  @include declaration('text-decoration', $drops--text-decoration);
-  @include declaration('border-width', $drops--border-width);
-  @include declaration('border-style', $drops--border-style);
-  @include declaration('border-color', $drops--border-color);
-  @include declaration('border-radius', $drops--border-radius);
-  @include declaration('color', $drops--color);
-  @include declaration('box-shadow', $drops--box-shadow);
-  @include declaration('background', $drops--background);
-  @include declaration('background-color', $drops--background-color);
-  @include declaration('min-width', $drops--min-width);
-  @include declaration('max-width', $drops--max-width);
-  @include declaration('width', $drops--width);
-  @include declaration('z-index', $drops--z-index);
-  @include declaration('-include-drops--gutter', $drops--margin);
-
-  @include block-generation($drop--selector);
-}
+$drop--selector: selector(new 'drop' with $drops--syntax);
+
+@include statement($drop--selector,(
+  (font-size, $drops--font-size),
+  (font-family, $drops--font-family),
+  (font-weight, $drops--font-weight),
+  (text-transform, $drops--text-transform),
+  (letter-spacing, $drops--letter-spacing),
+  (line-height, $drops--line-height),
+  (text-align, $drops--text-align),
+  (text-decoration, $drops--text-decoration),
+  (border-width, $drops--border-width),
+  (border-style, $drops--border-style),
+  (border-color, $drops--border-color),
+  (border-radius, $drops--border-radius),
+  (color, $drops--color),
+  (box-shadow, $drops--box-shadow),
+  (background, $drops--background),
+  (background-color, $drops--background-color),
+  (min-width, $drops--min-width),
+  (max-width, $drops--max-width),
+  (width, $drops--width),
+  (z-index, $drops--z-index),
+  ('-include-drops--gutter', $drops--margin),
```

Comment:
the syntax for mixins has changed with v4. It should be written in the way below
```scss
('@include drops--gutter', $drops--margin),
```